js/ui/handler/scroll_zoom.js

```diff
@@ -146,31 +151,31 @@ ScrollZoomHandler.prototype = {
 
 
 /**
- * Zoom start event. This event is emitted just before the map begins a transition from one
- * zoom level to another, either as a result of user interaction or the use of methods such as `Map#jumpTo`.
+ * Fired just before the map begins a transition from one zoom level to another,
+ * as the result of either user interaction or methods such as [`Map#jumpTo`](#Map#jumpTo).
```

Comment:
💭 Because `Map#jumpTo` is instantaneous, I don't think it triggers a `zoomstart` event. `Map#flyTo` is a better example.

My apologies -- I know this is tangential to the point of this PR.